runner/runner.go

```diff
@@ -1800,6 +1801,21 @@ retry:
 		builder.WriteRune(']')
 	}
 
+	var copyright string
+	if httpx.CanHaveTitleTag(resp.GetHeaderPart("Content-Type", ";")) {
+	    copyright = httpx.ExtractCopyright(resp)  // This will return a space-delimited string of years
```

Comment:
Why do we extract copyright text here and not under `scanopts.OutputCopyright` if block?

Reply:
I basically just copied the same functions that exist for Title extraction, but changed them to copyright instead. If there is a better way, or if I mis-copied that format from Title extraction, happy to mod.